src/verdant/panel/events/event-date.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from "react";
-import { connect } from "react-redux";
-import { verdantState } from "../../redux/index";
-import { Checkpoint } from "../../../lilgit/model/checkpoint";
-import { History } from "../../../lilgit/model/history";
-import NotebookEvent from "./event";
-import { dateOpen, dateClose, eventState } from "../../redux/events";
+import {connect} from "react-redux";
+import {verdantState} from "../../redux/index";
+import {Checkpoint, CheckpointType} from "../../../lilgit/model/checkpoint";
+import {History} from "../../../lilgit/model/history";
```

Comment:
Is this import used?

Reply:
Used in the interface for the accumulator.